app/templates/views/uploads/preview.html

```diff
@@ -27,8 +28,9 @@
           'main.send_uploaded_letter',
           service_id=current_service.id,
         )}}" class='page-footer'>
-        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-        <input type="hidden" name="file_id" value="{{ file_id }}" />
+          {{ form.csrf_token }}
+          {{ radios(form.postage, hide_legend=true, inline=True) }}
+          {{ form.file_id(value=file_id) }}
         <button type="submit" class="button">Send 1 letter</button>
```

Comment:
If you use the `page_footer` macro here you won’t need to manually print the CSRF token above.
```suggestion
            {{ page_footer('Send 1 letter') }}
```

Reply:
Oh nice. Have changed.